### ISSUE TITLE
[DQT] Saved Queries - get updated / overwritten without hitting `Save`

### ISSUE THREAD

**Issue by christinerogers** [aces/Loris#9316]
This may not be an issue in the upcoming New DQT -- best to verify esp with a TestUser. 
Also this point should be added to the Test Plan when fixed. 

To reproduce: 
- load a Saved Query.  Do not hit the purple Save button during the following: 
- go to (e.g.) Filters, change the Filter, e.g. new field or new value.   
- Run the query
- exit and reload the DQT.   Notice the Saved Query has been altered.

**Comment by christinerogers** [comment]
this still seems legit

**Comment by HenriRabalais** [comment]
@christinerogers does this apply to the new dataquery tool as well?

**Comment by christinerogers** [comment]
@HenriRabalais yes definitely.  i just tried it on Demo.loris.ca (Beta DQT) - Anything fields/filters you change after reloading a saved query, will alter the saved query. 
to replicate: i added fields (like date of confinement) and a filter to the first saved query, and tried to run it, did not attempt to save it. then go back to the (beta) dqt front page and see the query is already altered.  

Maybe this just needs a very visible caveat (e.g. `Caution: saved queries can be altered by any user without expressly saving these changes`), though a behaviour change (option to save /do not save altered query) would be ideal.

**Comment by HenriRabalais** [comment]
@christinerogers

I issued a fix for this [aces/Loris#11215](https://github.com/aces/Loris/pull/11215). loading a saved query handed the welcome page's own copy of the filters straight to the editor, so changing a filter changed the query shown in the list too. With the new PR, it gets its own copy.

I also added a section to the test plan for loading a saved query and changing it without saving, like you asked.
